crates/lsp-async-stub/Cargo.toml

```diff
@@ -3,7 +3,7 @@ authors = ["tamasfe"]
 description = "An LSP server stub for futures"
 edition = "2021"
 name = "lsp-async-stub"
-version = "0.6.2"
+version = "0.6.3"
```

Comment:
I don't think this is needed. There were no changes since 4bf7b536bcf03369ec9318000d3e88e1bbf994ac (which AFAICT is the last release).

Reply:
We need to release on CI.

Reply:
Sounds good.